### DISCUSSION TITLE
Help unRaid Setup - Could not tune channel. Please check your tuner or antenna.

### DISCUSSION THREAD

**Discussion by pivoivo** [chrisbenincasa/tunarr#532] [Questions & Help]
Hey there. I just tried using tunarr. First of all thanks for the development. But I still have some questions and errors and I tought maybe someone here could help me.

First question is: Is it possible to direct stream with tunarr? Or will it always transcode? If i read correctly direct-play should be enabled? If not, what do I need to do to prevent transcoding?

Second issue is, that I can not make tunarr/Channels work in Plex. I already set up a Test-Channel and was able to watch the content via the tunarr WebUI. I then added a DVR to Plex and it showed up with the right name of the channel and the planned schedule. But one I try to access/play it Plex starts "buffering" and after some seconds I always get the "Could not tune channel. Please check your tuner or antenna." error. Nothing else.
Maybe someone could help me with that.

Kind regards :)

**Comment by chrisbenincasa** [comment]
Hi! To answer your questions:

>  Is it possible to direct stream with tunarr? Or will it always transcode?

We've simplified a few things from DizqueTV here -- Tunarr will never invoke the transcoding in Plex, but will always transcode the original source to a normalized output. #444 and #452 can provide more context on the decision to remove direct stream and the Plex transcoder code, respectively. The TL;DR for the former is 1. forcing transcode solves an entire class of issues in terms of channel playback (both smoothness and reliability) and 2. is simplify required anyway to use a good chunk of the features in DTV/Tunarr (flex, watermarks, etc). 

> "Could not tune channel. Please check your tuner or antenna." error. Nothing else.

Happy to help you debug what is going on here, but I will need some server logs to see what's really happening. Preferably at the 'debug' level (this can be changed in your settings)

**Reply by chubby-cheese** [reply]
I'm getting this same error on a brand new set up, but I'm working with Windows. I can play the channels via the Tunarr web UI fine. I've enabled debug logs, but I'm not sure where they're stored.

**Comment by photo-copy** [comment]
With regard to tuning channels, you have to make sure you have your channels mapped. In Plex, go to Settings -> Live TV & DVR -> Click on the 3 Enabled (or how many ever channels you have) in your tuner. Here you can map the channels, and they should then show up for viewing.
